teps/0044-data-locality-and-pod-overhead-in-pipelines.md

```diff
@@ -188,6 +188,7 @@ some functionality would require changes to this code, and some functionality ma
 * Functionality that could be supported with current pod logic (e.g. by
   [translating a Pipeline directly to a TaskRun](#pipeline-executed-as-taskrun)):
   * Sequential tasks (specified using [`runAfter`](https://github.com/tektoncd/pipeline/blob/main/docs/pipelines.md#using-the-runafter-parameter))
+  * [Parallel tasks](https://github.com/tektoncd/pipeline/blob/main/docs/pipelines.md#configuring-the-task-execution-order)
```

Comment:
I've moved parallel tasks to "changes that won't require updating entrypoint" because the entrypoint already supports multiple waitfiles; it would just be a matter of updating the controller logic to determine what Tasks a given Task depends on based on a DAG.

Reply:
great catch!! i was definitely overcomplicating it in my head 🤣